site/design-notes/state-of-valhalla/03-vm-model.md

```diff
@@ -613,12 +622,28 @@ and that `I` is really (somehow) an exotic (`Q`-like) descriptor for
 `int.val`.  The `L` descriptor returns `int.ref.class`, while the
 exotic descriptor returns `int.val.class`.
 
+[ Which mirror defines the fields for apis like `getFields()`?]
```

Comment:
Discussed on today's call and the conclusion was that there is a single set of metadata shared by both mirrors.  So it doesn't matter which is used in reflective `getX` operations